tox.ini

```diff
@@ -3,12 +3,11 @@ envlist = py27
 
 [testenv:py27]
 deps =
-    manifestparser >= 1.1
+    pytest
```

Comment:
Put that in alphabetical order and we are fine. Sorry for not noticing that earlier.